.circleci/config.yml

```diff
@@ -358,6 +358,35 @@ jobs:
           command: |
             ./ops/scripts/ci-docker-tag-op-stack-release.sh <<parameters.registry>>/<<parameters.repo>> $CIRCLE_TAG $CIRCLE_SHA1
 
+  contracts-bedrock-build:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    resource_class: xlarge
+    steps:
+      - checkout
+      - check-changed:
+          patterns: contracts-bedrock,op-node
+      - run:
+          name: print forge version
+          command: forge --version
+          working_directory: packages/contracts-bedrock
+      - restore_cache:
+          name: Restore Forge Artifacts Cache
+          key: ctb-artifacts-cache-{{ checksum "packages/contracts-bedrock/semver-lock.json" }}
```

Comment:
Not all files are committed to in the `semver-lock.json` so this could hit the cache when it should actually miss

Reply:
Yeah this part hasn't been fixed up yet - the primary issue is that the `forge-artifacts` folder doesn't seem to be getting cached in its entirety. Other jobs that restore the cache still have to start from scratch on compilation.

Reply:
Other thing is that we should ideally be caching as many builds as we can - most PRs only change a few contracts, so ideally they can pick up on an older cache to reduce compilation times and only recompile what changed. The semver lock checksum isn't sufficient for this, but would be nice to figure out how to persist the cache relatively often.